src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,44 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        try {
+            $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
+            $composerLock = json_decode(file_get_contents($composerLockPath), true);
```

Comment:
But `file_get_contents` do not throw an exception if the file does not exists. So, this try/catch block does not work.